docs/AxeService.md

```diff
@@ -5,7 +5,8 @@ Terra toolkit automatically includes the wdio-axe-service which enhances a webdr
 
 Under the key `axe` in the wdio.conf.js you can pass a configuration object with the following structure:
 
-* **inject** - True if the axe script should be injected by the test running. Disable if axe is already included in the test files, which slightly speed up runs. Defaults to true.
```

Comment:
We should keep this documentation as a consumer can still set this to false if they wouldn't want to keep the default true.

Reply:
this was removed as was always defaulting to true as we were always ignoring the setting so it never worked

with the changes made we are still hardcoding it to default to true so it is working the same; therefore, removed it from the documentation as was never valid.

Reply:
well we should leave the documentation in place because the option should now actually work